### DISCUSSION TITLE
The classic wooden coaster should use the chainlift sound from RCT1

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenRCT2/OpenRCT2#18725] [Feature requests]
The classic wooden coaster should use the chainlift sound from RCT1, just to complete the "look" as it were. It uses the sound from RCT2 right now, and that just feels so wrong. I know this is incredibly minor, but it would still be a nice change, even if just for nostalgia's sake. For all I know, it would probably take less than a minute to change it.

**Comment by spacek531** [comment]
Is this something asset packs can solve?

**Reply by Derpercoaster** [reply]
I'm not sure. I don't know how asset packs work since they're so new. I haven't messed around with them yet.

**Reply by ZeeMaji** [reply]
I think asset packs should be able to solve this if you can reference audio files in the RCT2 data folder in them.
